qiita_pet/templates/study_base.html

```diff
@@ -15,12 +16,26 @@
 
   function fill_artifact(aid) {
     $("#study-main").html("ARTIFACT " + aid + " HTML STUFF");
+    $.get("/admin/artifact/", {artifact_id: +aid })
```

Comment:
The aid passed in is done using pure javascript on the client side before the get call, so yes it is correct. It is to make sure that what comes from the user side is a valid integer before the call.